### DISCUSSION TITLE
When would one use `StdevMethod` in a mosaic

### DISCUSSION THREAD

**Discussion by metasim** [cogeotiff/rio-tiler#512] [Q&A] [answered]
I'm curious to know when or why one would want to create a mosaic where the cell value is the standard deviation of the tile stack? All the other `MosaicMethodBase` classes make sense to me, except this one. 🤔

**Comment by vincentsarago** [comment, marked as answer]
👋 @metasim thanks for the question. To be honest I don't remember specifically why we added it but I know I've used it once to find places where the pixel values (e.g NDVI) was unstable over time! Maybe people won't use the method a lot but at least it gives another example of what you can do 😄

**Reply by metasim** [reply]
@vincentsarago Thanks for the context. Very helpful! :)
